packages/amplication-server/src/core/entity/entity.service.ts

```diff
@@ -2057,7 +2060,10 @@ export class EntityService {
       include: { entityVersion: true },
     });
 
-    if (isSystemDataType(field.dataType as EnumDataType)) {
+    if (
+      isSystemDataType(field.dataType as EnumDataType) &&
+      field.dataType !== "Id"
```

Comment:
since now we skip the current check, we should think about adding a new check specifically for a field of type "ID" - that the user is only changing the "idType" property and no other property